svcdiscovery/plugin.cfg

```diff
@@ -0,0 +1,8 @@
+svcdiscovery:svcdiscovery
+errors:errors
+health:health
+forward:forward
+cache:cache
+loop:loop
+reload:reload
+loadbalance:loadbalance
```

Comment:
Weird... their comment is less than optimal. I don't really understand what they mean by "feel the effects" - sounds more like something I'd read from a psychology book rather than a software component. As always, it's just easier to go read the source code to understand **how** the ordering here affects the DNS behaviour.

Reply:
Fixed with 93faaa3.